sklearn/feature_selection/_mutual_info.py

```diff
@@ -137,7 +137,7 @@ def _compute_mi_cd(c, d, n_neighbors):
 
     kd = KDTree(c)
     m_all = kd.query_radius(c, radius, count_only=True, return_distance=False)
-    m_all = np.array(m_all) - 1.0
+    m_all = np.array(m_all) 
```

Comment:
Was the intention to remove the + 1 from `digamma(m_all + 1)` a few lines below?

Reply:
Yes it was , because it was unnecessary increment

Reply:
This PR has not removed the `+ 1` yet a few lines below. May you push a commit to remove it?